soap/src/java/com/zimbra/soap/mail/type/IMAPItemInfo.java

```diff
@@ -0,0 +1,41 @@
+package com.zimbra.soap.mail.type;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+
+import com.zimbra.common.soap.MailConstants;
+
+@XmlAccessorType(XmlAccessType.NONE)
+public class IMAPItemInfo {
```

Comment:
We already have a class ImapMessageInfo that encapsulates this information, as well as the item type, flags, and tags. Either ImapMessageInfo should be made to subclass ImapItemInfo, or ImapMessageInfo can be used directly - in which case the other fields could be made optional.